scripts/environment.sh

```diff
@@ -61,22 +61,10 @@ export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export M3PATH=${M3PATH:-"${HOME}/go/src/github.com/metal3-io"}
 export CAPM3_LOCAL_IMAGE="${CAPM3PATH}"
 
+export PATH=$PATH:$HOME/.krew/bin
```

Comment:
is this needed?

Reply:
yeah for some reason when devenv install krew it will not be available in the path here ! I had to add it